Source/art/FTFontEnumerator.m

```diff
@@ -245,8 +245,7 @@ static void add_face(NSString *family, int family_weight,
       return;
     }
 
-  fi->displayName = [[family stringByAppendingString: @" "]
-			     stringByAppendingString: faceName];
+  fi->displayName = [[NSString stringWithFormat: @"%@ %@", family, faceName] retain];
```

Comment:
True enough, but as it was it's clear enough that this was just a bug that doesn't seem to have bitten anyone in an obvious way, since all -string* methods explicitly return autoreleased instances.